roles/kubernetes/master/tasks/kubeadm-upgrade.yml

```diff
@@ -3,7 +3,7 @@
   uri:
     url: "https://{{ ip | default(fallback_ips[inventory_hostname]) }}:{{ kube_apiserver_port }}/healthz"
     validate_certs: false
-  when: inventory_hostname == groups['kube-master']|first
+  when: inventory_hostname == groups['kube-master']
```

Comment:
@RickHaan this now never runs, `==` should be `in`

Reply:
You're right. Do you want me to create a PR to correct my error?

Reply:
already on it